package/upgrade/upgrade_manifests.sh

```diff
@@ -394,6 +394,16 @@ EOF
 
   upgrade_managed_chart_from_version $UPGRADE_PREVIOUS_VERSION harvester harvester.yaml
   NEW_VERSION=$REPO_HARVESTER_CHART_VERSION yq e '.spec.version = strenv(NEW_VERSION)' harvester.yaml -i
+
+  if [ "$UPGRADE_PREVIOUS_VERSION" == "v1.0.3" ]; then
+    yq e '.spec.values.longhorn.createStorageClass = true' -i harvester.yaml
+  fi
+
+  local sc=$(kubectl get sc -o json | jq '.items[] | select(.metadata.annotations."storageclass.kubernetes.io/is-default-class" == "true")')
```

Comment:
Do we only check any `is-default-class`?
Or do we also need to check the `default` SC is not `harvester-longhorn`?

Reply:
> Do we only check any `is-default-class`? Or do we also need to check the `default` SC is not `harvester-longhorn`?

please wait, I will rewrite in the next commit.